delta-inspect/Cargo.toml

```diff
@@ -17,4 +17,10 @@ env_logger = "0"
 [dependencies.deltalake]
 path = "../rust"
 version = "0"
-features = ["s3", "azure", "glue", "gcs"]
+features = ["azure", "gcs"]
```

Comment:
I think we should still keep the s3 and glue feature flag here?

Reply:
I stumbled across this as well, I think they are now enabled via the section right below this, but a bit more fine granular...